content/en/hosting-and-deployment/hosting-on-aws-amplify.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Host on AWS Amplify
+linktitle: Host on AWS Amplify
+description: AWS Amplify provides the fastest way to develop and deploy a mobile or web app. 
```

Comment:
This is very subjective and very ... advertising. Find a more neutral sentence to describe the service.

Reply:
I think so. How about

```diff
- AWS Amplify provides the fastest way to develop and deploy a mobile or web app.
+ AWS Amplify is a way to develop and deploy a mobile or web app.
```